src/rules/rrd/noInlineStyles.ts

```diff
@@ -0,0 +1,44 @@
+import type { SFCTemplateBlock } from '@vue/compiler-sfc'
+import getLineNumber from '../getLineNumber'
+import type { FileCheckResult, Offense } from '../../types'
+import { BG_RESET, BG_WARN, TEXT_INFO, TEXT_RESET, TEXT_WARN } from '../asceeCodes'
+
+const results: FileCheckResult[] = []
+
+const checkNoInlineStyles = (template: SFCTemplateBlock | null, filePath: string) => {
+  if (!template) {
+    return
+  }
+
+  const regex = /style\s*=\s*['"][^'"]*['"]/g
+
+  const matches = [...template.content.matchAll(regex)]
+
+  matches?.forEach((match) => {
+    const lineNumber = getLineNumber(template.content.trim(), match[0])
```

Comment:
please add the `from` parameter here otherwise if the same pattern found multiple times, it will report the same line number for them